src/util.js

```diff
@@ -481,6 +481,30 @@ export default {
   },
 
   /**
+   * Get native Web Cryptography api for old browsers only, including legacy
+   * implementations of the spec e.g IE11 and Safari 8/9. The default
+   * configuration is to use the api when available. But it can be deactivated
+   * with config.use_native
+   * @return {Object}   The SubtleCrypto api or 'undefined'
+   */
+  getWebCryptoPrefixed: function() {
+    if (!config.use_native) {
+      return;
+    }
+
+    var _self = this.getGlobal();
+    if (typeof _self === 'undefined') {
+      return;
+    }
+
+    if (typeof _self.crypto !== 'undefined') {
+      return _self.crypto.webkitSubtle;
```

Comment:
is this line correct? i think this will always make return _self.crypto.webkitSubtle whenever self.crypto is defined.

anyway i like this mixed approach just for legacy browsers, 

we should just get sure that on recent EDGE and Safari this does not make call always the old API that maybe is still there.

i would implement a check that the self.crypto is not there before the check for the legacy API.